src/Output/ProgressBar.php

```diff
@@ -63,10 +65,12 @@ class ProgressBar
      * Options for progress bar.
      */
     private array $options = [
-        'pointer'    => '>',
-        'loader'     => '=',
-        'color'      => 'white',
-        'labelColor' => 'white',
+        'pointer'        => '>',
+        'loader'         => '=',
+        'color'          => 'white',
+        'labelColor'     => 'white',
+        'labelPosition'  => 'bottom',   // position of the label according to the progress bar (one of 'top', 'bottom', 'left', 'right')
+        'showPercentage' => true,       // in spinner mode, you may not want to display the percentage of the progress because you don't know in advance how long the processing will take (during an asynchronous call for example)
```

Comment:
i think the comments can go to readme docs

Reply:
the comment regarding the `label Position` option has been removed and is already in the readme.
As for the one on the `showPercentage` option, I think that for the moment we should keep it because this option is only intended for internal use. If we decide to make it public, we can then remove it here and add it to the readme like the others